projects/plugins/jetpack-mu-wpcom/composer.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "automattic/jetpack-mu-wpcom",
+	"description": "Enhances your site with features powered by WordPress.com",
+	"type": "wordpress-plugin",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.4",
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-coverage": [
+			"php -dpcov.directory=. ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+		],
+		"test-php": [
+			"@composer phpunit"
+		],
+		"build-production": "echo 'Add your build step to composer.json, please!'",
+		"build-development": "echo 'Add your build step to composer.json, please!'"
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"mirror-repo": "Automattic/jetpack-mu-wpcom",
+		"release-branch-prefix": "jetpack-mu-wpcom",
```

Comment:
I wonder if we can get away with just "jetpack" here, if we'll always release it alongside Jetpack.

Reply:
The idea would be that we'd do the prerelease branch, do `jetpack release` for both plugins in step 2, and then only have to create the one release branch at the "Creating the Release Branch" step. Then if any cherry-picks are needed, we'd only have to cherry-pick them to the one release branch instead of both (unlike some of the stuff going on at p1670007480837609-slack-C01U2KGS2PQ and later where commits had to be cherry-picked to three different release branches).